library/Icingadb/Widget/ItemList/TicketLinkObjectList.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/* Icinga DB Web | (c) 2025 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget\ItemList;
+
+use Icinga\Exception\NotImplementedError;
+use Icinga\Module\Icingadb\Model\Comment;
+use Icinga\Module\Icingadb\Model\Downtime;
+use Icinga\Module\Icingadb\View\CommentRenderer;
+use Icinga\Module\Icingadb\View\DowntimeRenderer;
+use ipl\Orm\Model;
+use ipl\Web\Widget\ItemList;
+
+/**
+ * TicketLinkObjectList
+ *
+ * Create a list of icingadb objects with ticket links
+ *
+ * @template Item of Comment|Downtime
```

Comment:
Name this `Result`. Matches what the `ObjectList` and `ItemList` will use.

The `@extends` definition can continue using a single template. The second is optional, since here there is no distinction between *result* and *item*.